packages/core/src/tests/basic-navigation/content-width/index.html

```diff
@@ -49,7 +49,7 @@
           style="fill: url(#x05mhzd7ga)"
         />
       </svg>
-      <ix-menu>
+      <ix-menu expanded-navigation-menu-preferred>
```

Comment:
I think we should only set this for tests where it is actually needed to check functionalitly. Since the default changed we can adapt the screenshots.